store/types/store.go

```diff
@@ -277,6 +277,10 @@ type StoreKey interface {
 	String() string
 }
 
+// CapabilityKey represent the Cosmos SDK keys for object-capability
+// generation in the IBC protocol as defined in https://github.com/cosmos/ics/tree/master/spec/ics-005-port-allocation#data-structures
+type CapabilityKey StoreKey
```

Comment:
Isn't this just a renaming of the `StoreKey`? Is it completely necessary?

Reply:
Really we should call the `StoreKey`s `CapabilityKey`s, since that's what they are (nothing is specific to stores).

Reply:
I don't have a strong opinion about either. @alexanderbez thoughts here?